registry/vclab/star/container.yaml

```diff
@@ -0,0 +1,10 @@
+docker: tipsarin/star2.7.2a
```

Comment:
So if we want to be consistent with namespace, then the module folder for this container should be:

```bash
registry/tipsarin/star2.7.2a/container.yaml
```
E.g., if someone had the namespace of "vclab" on docker hub you shouldn't be able to claim it here. I also think that is why the test is failing - it's looking for vclab/star/ as a module (but per the container name it should match that).